release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -1086,15 +1061,15 @@ Hawkular is now deprecated and will be removed in a future release.
 
 [discrete]
 [[ocp-311-ocp-uses-registry-redhat-io]]
-==== New registry source for Red Hat images
+==== New Registry Source for Red Hat images
 
 Instead of `registry.access.redhat.com`, {product-title} now uses
 `registry.redhat.io` as the source of images for version 3.11. For access,
 `registry.redhat.io` requires credentials.
```

Comment:
This is linked to the appropriate topic in the registry section, but we could add a link here too. I'll leave that up to you  :smiley_cat: 

"See xref:../install_config/configuring_red_hat_registry.adoc#install-config-configuring-red-hat-registry[Authentication Enabled Red Hat Registry]
 for more information."